sap/note/note.go

```diff
@@ -79,6 +79,12 @@ func GetTuningOptions(saptuneTuningDir, thirdPartyTuningDir string) TuningOption
 			system.WarningLog("For more information refer to the man page saptune-migrate(7)")
 			continue
 		}
+		if !strings.HasSuffix(fileName, ".conf") {
```

Comment:
@angelabriel I would consider to refactor the ListDir actions like in https://github.com/SUSE/saptune/pull/19/files#diff-1776a94cd0cd10479ea4cc300c3794f6L69 and move it to some Util function, where you pass the dir and the message to log in case of error and this return you the file list. This way, we avoid to include the same if in all the places where you need to test if there was an error on the file list or not.
Also, I think it will improve the results on CodeClimate and solve the problem it is currently pointing.

Reply:
changes done and CodeClimate is green again.